### PR TITLE
feat(react-native): :sparkles: types for react-native-modern-datepicker

### PR DESCRIPTION
**Pull request by Ankan002**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test react-native-modern-datepicker`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/react-native-modern-datepicker/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        daysAnimationDistance?: number;
+    };
+}
+
+export default function(props: ModernDatepickerProps): JSX.Element;
+export function getToday(): string;
+export function getMonthName(month: number): string;
+export function getFormatedDate(date?: Date, format?: string): string;
+export function toPersianNumber(value: string): string;
+export function toEnglish(value: string): string;
```
Comment (peterblazejewicz) [inline types/react-native-modern-datepicker/index.d.ts:46]:
```suggestion
export function getToday(): string;
export function getFormatedDate(date?: Date, format?: string): string;
```
there are named exports in the module file, only couple of symbols are being exported IMO:
https://github.com/HosseinShabani/react-native-modern-datepicker/blob/master/src/index.js#L4-L5

Comment (Ankan002) [reply types/react-native-modern-datepicker/index.d.ts:46]: Yeah that's right my fault will rectify and  rectify this PR. Actually I misundertood the utils file. Sorry for that.

Comment (Ankan002) [reply types/react-native-modern-datepicker/index.d.ts]: Hey @peterblazejewicz I have just now pushed the required changes to fix the issue of extra functions. Please review once you have time.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@Ankan002 congrats for your first contribution!

**Comment by Ankan002** [comment]
Ready to merge

**Comment by Ankan002** [comment]
Ready to merge

**Pull request merged**